### DISCUSSION TITLE
Bank Organizer Mode

### DISCUSSION THREAD

**Discussion by CIeetZ** [runelite/runelite#16645] [Ideas]
Hello. I just had an idea to help us organize our banks faster. When "Bank Organizer" plugin is activated, when You search for an item , instead of only showing that item that you search for , it points an arrow towards where it is in Your bank. This will make organizing the bank way faster, as currently the only way to know where an item is in a huge messy bank, is to manually look for it. This can take a minute sometimes and is a big time waster, making organizing an entire bank take forever. When the arrow points towards it we can quickly know where stuff is and get to sorting conveniently. The only other method of knowing where something is quickly is to 1. withdraw it all, 2. remove the placeholder, 3. redeposit it , and then you know its in the last slot in ur main tab. Who wants to do that again and again? Thanks for Your time.

**Comment by LlemonDuck** [comment]
Why would an arrow to where it is be better than filtering to exactly what you want?

If you're trying to sort your bank to get rid of the clutter, search for it, move it to another tab, and then have a clean bank from then on.
